pkgs/development/python-modules/beanquery/default.nix

```diff
@@ -7,18 +7,18 @@
   python-dateutil,
   pytestCheckHook,
   setuptools,
-  tatsu,
+  tatsu-lts,
 }:
-buildPythonPackage rec {
+buildPythonPackage {
   pname = "beanquery";
-  version = "0.1.0";
+  version = "0.1.0-unstable-2025-01-10";
```

Comment:
Can we ask upstream to cut a release?

Reply:
> Can we ask upstream to cut a release?

Someone already did at https://github.com/beancount/beanquery/issues/226#issuecomment-2614649306

Upstream suggested to use version 0.1.0 that requires `tatsu >= 5.7.4, < 5.8.0` which is not present at nixpkgs, using `pythonRelaxDepsHook` doesn't work, because beanquery fails to build with original tatsu v5.13.1 (non-LTS one).

Reply:
You can use `substituteInPlace` to replace tatsu with tatsu-lts.

Reply:
> You can use `substituteInPlace` to replace tatsu with tatsu-lts.

Wouldn't help because there are API changes in `tatsu` v5.13.0 and later that are present in lts version as well. These change cause all `beanquery` tests to fail with `AttributeError`. The only way to package `beanquery` at `v0.1.0` is to pin `tatsu` dependency at previous version `v5.12.2`